projects/demo/src/app/cds-theme-select.component.ts

```diff
@@ -35,7 +35,13 @@ export class CdsThemeSelectComponent implements OnInit, OnDestroy {
     this.applyTheme(null);
   }
 
-  applyTheme(theme: string) {
+  applyTheme(event: Event) {
```

Comment:
This breaks the `ngOnInit`.

Reply:
Good point. I made the change cause it was causing a runtime error and I did the changes to unblock me on the rest. I'll fix it tomorrow

Reply:
I would suggest this:

```typescript
onThemeChange(event: Event) {
  if (event?.target) {
    this.applyTheme(event.target.value); // may have to cast `target` to `any` here
  }
}

private applyTheme(theme: string) {
  ...
}